decidim-core/app/packs/src/decidim/external_domain_warning.js

```diff
@@ -1,5 +1,13 @@
 /* eslint-disable require-jsdoc */
 
+const EXCLUDE_CLASSES = [
+  "ssb-icon"
+];
+
```

Comment:
I'm not 100% sure if we want this approach, as its based on the `ExternalLink` deprecated approach and with the redesign we're actually using `data-external-link="false"` for a similar same problem, so maybe we want to change this to `data-external-domain-warning="false"` for consistency?

Reply:
I think it's quite cleaner `data-external-domain-warning="false"`. When you invoke this function, you may use it like this:
```
  element.querySelectorAll("a[target=\"_blank\"]:not([data-external-link=\"false\"])").forEach((elem) => new ExternalLink(elem))
```

In that way you don't worry about clases or elements. You only initialize that modal on the elements you want.

Reply:
I (finally) implemented it. It went a bit out of hand as in the process I've also changed it from jQuery to vanilla Javascript, and I also wanted to make it consistent with the [new ExternalLink class](https://github.com/decidim/decidim/tree/develop/decidim-core/app/packs/src/decidim/redesigned_external_link.js).

You can give it an eye if you want (and have the availability) @Crashillo